CONTRIBUTING.md

```diff
@@ -10,7 +10,7 @@ When in doubt, keep your pull requests small. To give a PR the best chance of ge
 
 As with issues, please begin the title with [ComponentName].
 
-When adding new features or modifying existing, please attempt to include tests to confirm the new behaviour. You can read more about our test setup [here](https://github.com/mui-org/material-ui/blob/master/test/README.md).
```

Comment:
😆

Reply:
I thought I better check them all 😉

Reply:
I'm not sure what to change this one to though https://github.com/mui-org/material-ui/blob/master/packages/material-ui-icons/README.md#upgrading

Reply:
We have built a jscodeshift codemod to help you upgrade.

Reply:
With the link on upgrade?